.github/workflows/test.yml

```diff
@@ -30,10 +31,9 @@ jobs:
         python -m pip install types-PyYAML
         python -m pip install types-requests
         python -m pip install pytest
-        python -m pip install aicli
         python -m pip install paramiko==2.12.0
         python -m pip install gitpython
-        python -m pip install kubernetes
+        python -m pip install -r requirements.txt
```

Comment:
This didn't work for me in the past. Hence, we have `cat requirements.txt  | xargs -n1 $PYTHON_CMD -m pip install` instead of this. Are you `-r reqs.txt` will work?

Reply:
Seems so. The github action passes. The  "-r" option is documented by `pip`.

So at least the pip versions used on github actions (ie. the relevant versions) support this.

Reply:
```
$ pip install --help | grep -C4 -e '-r\>'

Usage:   
  pip install [options] <requirement specifier> [package-index-options] ...
  pip install [options] -r <requirements file> [package-index-options] ...
  pip install [options] [-e] <vcs project url> ...
  pip install [options] [-e] <local project path> ...
  pip install [options] <archive url/path> ...

--
  pip also supports installing from "requirements files", which provide
  an easy way to specify a whole environment to be installed.

Install Options:
  -r, --requirement <file>    Install from the given requirements file. This
                              option can be used multiple times.
  -c, --constraint <file>     Constrain versions using the given constraints
                              file. This option can be used multiple times.
  --no-deps                   Don't install package dependencies.
```

Reply:
Yes, I've tried -r initially, but I had to resort to xargs. I'm wondering if we need to do that here too.

Reply:
As said, we don't.

Note how the github actions pass -- which is all that matters to show that it's supported in the environment we care about.